actions/push/README.md

```diff
@@ -0,0 +1,87 @@
+# Flux Push GitHub Action
+
+This GitHub Action can be used to push OCI artifacts for usage with Flux.
+
+## Usage
+
+Example workflow for pushing the production artifacts:
+
+```yaml
+name: Push Production Artifacts
+
+on:
+  push:
+    branches: [production]
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write # for pushing
+      id-token: write # for signing
+    steps:
+    - uses: actions/checkout@v4
+    - uses: controlplaneio-fluxcd/distribution/actions/setup@main
+    - uses: sigstore/cosign-installer@v3
+    - uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - id: inputs
+      run: |
+        echo "source=$(git config --get remote.origin.url)" >> $GITHUB_OUTPUT
+        echo "revision=$(git branch --show-current)@sha1:$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
+    - uses: controlplaneio-fluxcd/distribution/actions/push@main
+      id: push
+      with:
+        repository: ghcr.io/${{ github.repository }}
+        path: clusters
+        source: ${{ steps.inputs.outputs.source }}
+        revision: ${{ steps.inputs.outputs.revision }}
```

Comment:
This shouldn't be required, we should be able to generate these settings and avoid users messing up the revision format.

Reply:
Fixed, thanks!